tox.ini

```diff
@@ -69,3 +64,12 @@ commands = bash {toxinidir}/tools/test/functest.sh
 
 [testenv:docs]
 commands = sphinx-build -j auto -d {toxinidir}/doc/build/doctrees -b html {toxinidir}/doc/source doc/build/html
+
+[testenv:coverage]
+commands =
+    coverage erase
```

Comment:
Do you need to have an stestr line here to run some tests? I'm just going by the neutron section

Reply:
I think it's not necessary to do so since we're overriding the PYTHON env variable to capture the coverage data. But still, I think we need to move the "coverage erase" line above the "stestr run" in order to ensure that we don't lose the coverage data captured by the previous run. 

Also, we need to introduce a "depends = py3" here so tox can determine the run order when i.e. "--parallel" is given.